### ISSUE TITLE
Installation error

### ISSUE THREAD

**Issue by regexgit** [nikrou/phyxo#36]
On web installation:

`Fatal error: Uncaught Symfony\Component\Dotenv\Exception\PathException: Unable to read the "/home/clients/89bc9bd72e54950241bd9b1e083c63a4/phyxo/.env" environment file. in /home/clients/89bc9bd72e54950241bd9b1e083c63a4/phyxo/vendor/symfony/dotenv/Dotenv.php:465Stack trace: #0 /home/clients/89bc9bd72e54950241bd9b1e083c63a4/phyxo/vendor/symfony/dotenv/Dotenv.php(50): Symfony\Component\Dotenv\Dotenv->doLoad(false, Array) #1 /home/clients/89bc9bd72e54950241bd9b1e083c63a4/phyxo/index.php(24): Symfony\Component\Dotenv\Dotenv->load('/home/clients/8...') #2 {main} thrown in /home/clients/89bc9bd72e54950241bd9b1e083c63a4/phyxo/vendor/symfony/dotenv/Dotenv.php on line 465`

=> no file ".env"

**Comment by regexgit** [comment]
**Installation procédure**

OK now but it's a little messy: at the first installation I followed the procedure "Manual" with the package "phyxo-1.10.1.zip" in which there was the directory "vendor" with all the dependencies.

For the second installation I realized that the package "phyxo_master" was much smaller, and for good reason, it did not contain "vendor". So I used the directory "vendor" of the first install but it did not work (error Attempt to load class "WebProfilerBundle" from namespace "Symfony\Bundle\WebProfilerBundle")

So to properly re-install the dependencies I ran "composer install", everything ok but on the web interface I still had a flop of error messages. On the other hand the command `./bin/console phyxo: install` went well, I could enter all the information of the database.

So for the moment the procedure "Manual" (whose name is poorly chosen by the way, I would have seen better "Auto") does not work, it is necessary to use "From source".

**Execution**

When I connect to my site:
```
The page is not redirected correctly
Firefox has detected that the server is redirecting the request for this address in a way that will not succeed.
```

I also installed locally on my PC using `symfony serve`: exactly same, looking at the log server one realizes that it loops in the redirections:
```
[Sat Oct 5 23:21:59 2019] 127.0.0.1:51066 [302]: /
[Sat Oct 5 23:21:59 2019] 127.0.0.1:51067 [302]: / identification
[Sat Oct 5 23:21:59 2019] 127.0.0.1:51068 [302]: / identification
...
```
